galley/cmd/galley/cmd/server.go

```diff
@@ -140,9 +140,11 @@ func serverCmd() *cobra.Command {
 	svr.PersistentFlags().StringVar(&serverArgs.DomainSuffix, "domain", serverArgs.DomainSuffix,
 		"DNS domain suffix")
 	svr.PersistentFlags().BoolVar(&serverArgs.DisableResourceReadyCheck, "disableResourceReadyCheck", serverArgs.DisableResourceReadyCheck,
-		"(DEPRECATED) Disable resource readiness checks. This allows Galley to start if not all resource types are supported")
+		"Disable resource readiness checks. This allows Galley to start if not all resource types are supported")
+	_ = svr.PersistentFlags().MarkDeprecated("disableResourceReadyCheck", "")
 	svr.PersistentFlags().StringSliceVar(&serverArgs.ExcludedResourceKinds, "excludedResourceKinds",
-		serverArgs.ExcludedResourceKinds, "(DEPRECATED) Comma-separated list of resource kinds that should not generate source events")
+		serverArgs.ExcludedResourceKinds, "Comma-separated list of resource kinds that should not generate source events")
+	_ = svr.PersistentFlags().MarkDeprecated("excludeResaourceKinds", "")
```

Comment:
Typo: excludeResaourceKinds should be excludedResourceKinds

Reply:
Whoops. Fixed.

Reply:
Did you remember to push the fix? I don't see it...

Reply:
Actually, it isn't! :)

excludedResaourceKinds
excludedResourceKinds